### PR TITLE
docs: Update about XboxDev & cxbx-r in README.md

### PR DESCRIPTION
**Pull request by Teufelchen1**
Hi! 👋 

Just minor adjustments suggested by @JayFoxRox. 

- Referencing XboxDev in the readme, to point interested people towards the community
- ~~Stating the incompatibility with cxbx-r to prevent issues such as [Samples don't work in CXBX-Reloaded #352](https://github.com/XboxDev/nxdk/issues/352) in the future~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 nxdk - *the new open source xdk*
 ================================
-nxdk is a software development kit for the original Xbox. nxdk is a revitalization of [OpenXDK](https://web.archive.org/web/20170624051336/http://openxdk.sourceforge.net:80/).
+nxdk is a software development kit for the original Xbox. nxdk is a revitalization of [OpenXDK](https://web.archive.org/web/20170624051336/http://openxdk.sourceforge.net:80/). It is maintained by the [XboxDev](https://github.com/XboxDev/XboxDev) community.
```
Comment (JayFoxRox) [inline README.md:3]: nit: I think I'd move this to a separate line to make editing easier

Comment (JayFoxRox) [inline README.md:3]: In addition to this, we should probably add a note to the wiki, explicitly recommending the Discord channel.

File: README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ You will need the following tools:
 - [CMake](https://cmake.org/)
 
 OS-specific instructions for installing these prerequisites can be found in the [Wiki](https://github.com/XboxDev/nxdk/wiki/Install-the-Prerequisites)
+Note that homebrew software build with nxdk won't work with [Cxbx-Reloaded](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded) because they don't support nxdks API.
```
Comment (JayFoxRox) [inline README.md:29]: I feel like this isn't really useful here?

It's burried in other information and part of "Getting Started" with zero-context. Some people might not even know what Cxbx-Reloaded is.

I feel this should be integrated in the wiki instead; in fact, the wiki incorrectly *mentions* Cxbx-Reloaded as functional solution for nxdk.

I'm not sure *where* to put this kind of information in the wiki though.

File: README.md (hunk 1 above)
Comment (Teufelchen1) [reply README.md]: New line added

File: README.md (hunk 2 above)
Comment (Teufelchen1) [reply README.md]: ftr, was based on a misunderstanding. The mentioning of cxbx-r in the wiki got removed.

**Pull request merged**